astropy/io/fits/column.py

```diff
@@ -461,7 +461,9 @@ def __init__(self, name=None, format=None, unit=None, null=None,
                  array=None, ascii=None):
         """
         Construct a `Column` by specifying attributes.  All attributes
-        except `format` can be optional.
+        except format can be optional, see :ref:`column_creation` and
```

Comment:
(This is a real nitpick, but please also change the comma after "optional" to a semi-colon.)